infrastructure/main.tf

```diff
@@ -102,7 +103,7 @@ module "ccd-data-store-api" {
 
     CCD_DEFAULTPRINTURL                 = "${local.default_print_url}"
 
-    ELASTIC_SEARCH_HOSTS                = "${format("http://%s:9200", data.azurerm_key_vault_secret.ccd_elastic_search_url.value)}"
+    ELASTIC_SEARCH_HOSTS                = "${var.elastic_search_enabled == "false" ? "" : "${format("http://%s:9200", join("", data.azurerm_key_vault_secret.ccd_elastic_search_url.*.value))}"}"
```

Comment:
horrible hack with the join. But terraform ternary operator is broken and otherwise would always evaluate both of the sides